modules/signoz/templates/values.yaml

```diff
@@ -1767,6 +1806,7 @@ otelCollector:
 
     autoscalingTemplate: []
     keda:
+      annotations:
```

Comment:
is this value being empty intended?

Reply:
It would probably cause issues. But also we are not using this, so install doesn't complain